notebooks/03_raster_zonal_stats.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        aggregation: Dict[  # A dict specifying the aggregation. See <code>create_zonal_stats</code> from the <code>geowrangler.vector_zonal_stats</code> module for more details
</pre><p>I'd just suggest to rename this to <code>aggregations</code> so that it's consistent with the vector zonal stats counterpart.</p><p> </p><p>Also, not urgent, but there might be an opportunity to rename the functions across both for consistency. Right now, it's:</p><ul><li><code>vector_zonal_stats.create_zonal_stats()</code> for vectors</li><li><code>raster_zonal_stats.create_raster_zonal_stats()</code> for rasters  </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/104/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
The reason is that unlike vector zonal stats, its not a list of aggregations -- its only 1 aggregation since normally rasters have only 1 band of data. 

As for the 'column' its actually not a property of the raster -- its just used as a default '{column_name}_{func}' if no 'output' (column name) is specified.

In hindsight, yes it should be named 
`create_vector_zonal_stats` and `create_raster_zonal_stats` (there's another method called `create_bingtile_zonal_stats`) for consistency. 

We should create a  separate ticket for that -- we should follow a deprecation policy as a template for these type of changes.

Reply:
Ahh I see. Actually, there are some rasters that have multiple bands (e.g. LANDSAT, MODIS), and sometimes we need to perform band math on them (such as the [computation of NDVI](https://www.usgs.gov/landsat-missions/landsat-normalized-difference-vegetation-index)) . 

But the current implementation can already be useful for single-band rasters, esp. the ones we'll likely use in poverty mapping (i.e. HRSL, night time lights). Maybe we can also just create a separate issue for multi-band support? In the meantime though, should we just add a note somewhere that the function only supports single bands for now?